daemon/gluster-blockd.c

```diff
@@ -463,15 +470,19 @@ main (int argc, char **argv)
     exit(errnosv);
   }
 
-  errnosv = blockNodeSanityCheck();
-  if (errnosv) {
-    exit(errnosv);
-  }
+  if (!gbConf.ignoreChecks) {
+    errnosv = blockNodeSanityCheck();
+    if (errnosv) {
+      exit(errnosv);
+    }
 
-  if (initDaemonCapabilities()) {
-    exit(EXIT_FAILURE);
+    if (initDaemonCapabilities()) {
```

Comment:
what is initDaemonCapabilities and is it not required during genconfig?

Reply:
It just checks for Capabilities of local  gluster-blockd daemon. Its required when gluster-blockd communicates with remote gluster-blockd's

So not needed for our use-case.